cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/AbstractStorageProviderNamespaceAdmin.java

```diff
@@ -69,9 +69,7 @@ public void create(NamespaceMeta namespaceMeta) throws IOException, ExploreExcep
 
     createLocation(namespaceMeta);
 
-    // only create non-default namespaces in Hive
-    if (cConf.getBoolean(Constants.Explore.EXPLORE_ENABLED) &&
-      !NamespaceId.DEFAULT.equals(namespaceMeta.getNamespaceId())) {
```

Comment:
There was already logic in BaseHiveExploreService to make default namespace create a no-op:
https://github.com/caskdata/cdap/blob/release/3.5/cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java#L756-L758

I'm just removing this code. It was only recently brought in by:
https://github.com/caskdata/cdap/pull/6441